src/battle_main.c

```diff
@@ -4984,7 +4984,14 @@ void SetTypeBeforeUsingMove(u16 move, u8 battlerAtk)
     }
     else if (gBattleMoves[move].effect == EFFECT_JUDGMENT)
     {
-        // TODO:
+        #ifdef IS_ITEM_PLATE
+            if (!(IS_ITEM_PLATE(gBattleMons[battlerAtk].item)))
```

Comment:
rather than have a hacky macro like this, define a new hold effect like `HOLD_EFFECT_PLATE` and check that instead